addons/website_sale/static/src/js/website_sale.js

```diff
@@ -140,7 +140,7 @@ publicWidget.registry.websiteSaleCartLink = publicWidget.Widget.extend({
         if ('website_sale_cart_quantity' in sessionStorage) {
             this.cartQty = sessionStorage.getItem('website_sale_cart_quantity');
         }
-        if (this.el.querySelector('.my_cart_quantity').innerText != this.cartQty) {
+        if (this.el.querySelector('.my_cart_quantity') && this.el.querySelector('.my_cart_quantity').innerText != this.cartQty) {
```

Comment:
Starting saas-16.1, we can use the optional chaining operator to simplify this kind of code.
```suggestion
        if (this.el.querySelector('.my_cart_quantity')?.innerText != this.cartQty) {
```